src/nls/root/strings.json

```diff
@@ -379,7 +379,9 @@
         "TITLE_SCALE": "Scale",
         "TITLE_SUFFIX": "Suffix",
         "TITLE_SETTINGS": "Settings",
-        "EXPORT_DOCUMENT_FILENAME": "document"
+        "EXPORT_DOCUMENT_FILENAME": "document",
+        "ONLY_UNSUPPORTED_LAYERS_SELECTED": "Unsupported Layer",
+        "NO_ASSETS": "Nothing configured"
```

Comment:
At the very least I'd change this to "No exports configured." But, ideally, this text would also explain to the user _how_ to configure some exports. I'm unsure about how to word this in a completely satisfactory way, though.